ksqldb-serde/src/main/java/io/confluent/ksql/serde/protobuf/ProtobufFormat.java

```diff
@@ -59,7 +59,7 @@ public String name() {
   protected ConnectSchemaTranslator getConnectSchemaTranslator(
       final Map<String, String> formatProps
   ) {
-    FormatProperties.validateProperties(name(), formatProps, ImmutableSet.of());
+    FormatProperties.validateProperties(name(), formatProps, getSupportedProperties());
```

Comment:
This is a fix from a previous PR, unrelated to this one, right?

Reply:
Correct